Framework/Crystal/src/SaveLauenorm.cpp

```diff
@@ -89,7 +97,12 @@ void SaveLauenorm::exec() {
   double minIntensity = getProperty("MinIntensity");
   int widthBorder = getProperty("WidthBorder");
   bool newFormat = getProperty("LaueScaleFormat");
-
+  std::string cellType = getProperty("CrystalSystem");
+  auto iter = std::find(m_typeList.begin(), m_typeList.end(), cellType);
+  long int cellNo = 1 + std::distance(m_typeList.begin(), iter);
+  std::string cent = getProperty("Centering");
+  auto iter2 = std::find(m_centeringList.begin(), m_centeringList.end(), cent);
+  long int centerNo = 1 + std::distance(m_centeringList.begin(), iter2);
```

Comment:
I think you could just use `auto centerNo` here to avoid the compiler warning.

Reply:
Made these changes